src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -79,6 +80,7 @@
 import org.kohsuke.stapler.StaplerRequest;
 import org.kohsuke.stapler.export.Exported;
 
+import javax.annotation.Nonnull;
```

Comment:
No jsr-305 annotations please.  We're using spotbugs annotations
```suggestion
import edu.umd.cs.findbugs.annotations.NonNull;
```